.github/workflows/verify_docker_image.yml

```diff
@@ -1,22 +1,88 @@
 name: Pull and Verify Docker Image
 
 on:
-  workflow_call:
   workflow_dispatch:
+    inputs:
+      amdgpu_target:
+        description: "AMDGPU target to verify"
+        required: true
+        default: "gfx942"
+        type: string
+      image_digest:
+        description: "Digest of image to verify"
+        required: true
+        type: string
+      runner_type:
+        description: "Runner type to use"
+        required: true
+        default: "linux-mi300-1gpu-ossci-rocm"
```

Comment:
Should the `runner_type` and `amdgpu_target` be coupled checked? Otherwise one can try to run on a runner without the correct GPU which should fail. Nothing to address right now but maybe something to keep in mind for a follow up.